lib/api_hammer/faraday/request_logger.rb

```diff
@@ -76,7 +81,7 @@ def call(request_env)
             'processing' => {
               'began_at' => began_at.utc.to_f,
               'duration' => now - began_at,
-              'activesupport_tagged_logging_tags' => @log_tags,
+              'activesupport_tagged_logging_tags' => log_tags,
```

Comment:
this is set local, should not be ivar or method. must have been a bug I had it as ivar before.